app/src/main/java/com/sun/premierleague/ui/team/TeamFragment.kt

```diff
@@ -0,0 +1,135 @@
+        val repository = RepositoryFactory.getRepository(context)
+        presenter = TeamPresenter(this, repository)
+    }
+
+    private fun initCollapsingToolbar() {
+        val collapsingToolbarLayout =
+            collapsingToolbarTeam as CollapsingToolbarLayout
+        val appBarLayout = appBarTeam as AppBarLayout
+        var isShow = true
+        var scrollRange = -1
+        appBarLayout.addOnOffsetChangedListener(AppBarLayout.OnOffsetChangedListener { barLayout, verticalOffset ->
+            if (scrollRange == -1) {
+                scrollRange = barLayout?.totalScrollRange!!
+            }
+            if (scrollRange + verticalOffset == 0) {
+                collapsingToolbarLayout.title = textDetailTeam.text
+                isShow = true
+            } else if (isShow) {
+                collapsingToolbarLayout.title = " "
+                isShow = false
+            }
+        })
+    }
+
+    override fun showTeam(data: List<TeamItem>) {
+        textDetailTeam.text = data.first().teamName
+        imageDetailTeam.loadImage(data.first().teamBadge)
+    }
+
+
+    override fun showCoach(coache: List<Coache>) {
+        if (coache.isEmpty()) {
+            view?.context?.showToast(R.string.err_no_coach)
+            return
+        }
+        val coachOne = coache.first()
+        textNameCoachOne.text = coachOne.coachName
+        textInfoCoachOne.text = String.format(
+            getString(R.string.text_info_coach_format),
+            coachOne.coachAge,
+            coachOne.coachCountry
+        )
+        if (coache.size > 1) {
+            val coachTwo = coache[1]
+            textNameCoachTwo.visibility = View.VISIBLE
+            textInfoCoachTwo.visibility = View.VISIBLE
+            textNameCoachTwo.text = coachTwo.coachName
+            textInfoCoachTwo.text = String.format(
+                getString(R.string.text_info_coach_format),
+                coachTwo.coachAge,
+                coachTwo.coachCountry
+            )
+        } else {
+            imageCoachTwo.visibility = View.INVISIBLE
+            textNameCoachTwo.visibility = View.INVISIBLE
+            textInfoCoachTwo.visibility = View.INVISIBLE
+        }
+    }
+
+    override fun showGoalkeeper(data: List<Player>) {
+        adapterGoalkeeper.replaceData(data)
+        recyclerGoalkeeper.scrollToPosition(0)
+    }
+
+    override fun showDefender(data: List<Player>) {
+        adapterDefender.replaceData(data)
+        recyclerPlayerDefend.scrollToPosition(0)
+    }
+
+    override fun showMidfielder(data: List<Player>) {
+        adapterMidfielder.replaceData(data)
+        recyclerPlayerMidfield.scrollToPosition(0)
+    }
+
+    override fun showForwarder(data: List<Player>) {
+        adapterForwarder.replaceData(data)
+        recyclerPlayerForward.scrollToPosition(0)
+    }
+
+    override fun showError(error: Any) {
+        view?.context?.showToast(error)
+    }
+
+    companion object {
+        private const val ID_TEAM = "idTeam"
```

Comment:
Đặt tên theo conventions nhé 
```suggestion
        private const val BUNDLE_ID_TEAM = "BUNDLE_ID_TEAM"
```